bodhi/server/push.py

```diff
@@ -47,6 +47,9 @@
 @click.option('--resume', help='Resume one or more previously failed pushes',
               is_flag=True, default=False)
 def push(username, cert_prefix, config, **kwargs):
+    if not username:
+        raise Exception("Please provide --username")
```

Comment:
Alternatively, we could make it an argument instead of an option, but I actually kinda like the prompt workflow myself. What do you think?